README.md

```diff
@@ -1,60 +1,114 @@
 
-## Installation
+Source code annotation tool offers an UI to annotate source code and review these annotations, and a CLI to define the code to be annotated and export the annotations.
 
-### Github OAuth tokens
+### Why is is needed?
 
-First you need OAuth application on github. [Read how to create it](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+In order to evaluate quality of ML models, as well as to create “ImageNet for Source Code” there is a need for tools to automate the data collection/labeling/annotation.
 
-On a [page](https://github.com/settings/developers) with your application you will need `Client ID` and `Client Secret`
+![Screenshot](.github/screenshot.png?raw=true)
 
-Copy `.env.tpl` to `.env` and set tokens there.
+## Requirements
 
-### Docker
+### Global dependencies
 
-```bash
-docker build -t srcd/code-annotation .
-docker run --env-file .env --rm -p 8080:8080 srcd/code-annotation
+You should already have [Go installed](https://golang.org/doc/install#install), and properly [configured the $GOPATH](https://github.com/golang/go/wiki/SettingGOPATH)
+
+```
+go version; # prints your go version
+echo $GOPATH; # prints your $GOPATH path
 ```
 
-### Non-docker
+The project must be under the `$GOPATH`, following the Go import conventions, what means you can go to its directory running:
 
-```bash
-go get github.com/src-d/code-annotation/...
-cd $GOPATH/github.com/src-d/code-annotation
-make serve
+```
+cd $GOPATH/src/github.com/src-d/landing
 ```
 
-## Development
-
-Backend:
+You need also [Yarn v1.x.x installed](https://yarnpkg.com/en/docs/install)
 
 ```
-make gorun
+yarn --version; # prints your Yarn version
 ```
 
-Frontend:
+### Github OAuth tokens
+
+1. You need an OAuth application on GitHub. See [how to create OAuth applications on GitHub](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
 
-If you want to benifit from frontend hot reloading feature this line in your `.env` file:
+    `Authorization callback URL: http://127.0.0.1:8080/oauth-callback`
 
+2. Copy `.env.tpl` to `.env`.
+
+3. Retrieve the values for your application's Client ID and Client Secret from the [GitHub Developer Settings page](https://github.com/settings/developers) and add them to the end of the corresponding lines in .env.
+
+## Installation
+
+You need to satisfy all [project requirements](#requirements), and then run:
+
+```bash
+$ go get github.com/src-d/code-annotation/...
+$ cd $GOPATH/github.com/src-d/code-annotation
+$ make serve
 ```
-UI_DOMAIN=http://127.0.0.1:3000
+
+It runs everything you need to get the tool working at [http://localhost:8080](http://localhost:8080)
+
+## Importing and Exporting Data
+
+### Import File Pairs for Annotation
+
+The file pairs must be initially provided via an [SQLite](https://sqlite.org/) database. The database **must follow the expected schema**, please [follow this link](./cli/examples/import/example.sql) to see an example.
```

Comment:
Drop the "initially", it implies that should have happened before this step.

Also since the schema is so important you should make it explicit and maybe paste the result of calling `DESCRIBE TABLE` here.

Reply:
About the schema: it was actually removed as requested, see https://github.com/src-d/code-annotation/pull/29#discussion_r163947646.
Pinging @bzz.